Validation/MtdValidation/plugins/BtlLocalRecoValidation.cc

```diff
@@ -183,11 +192,18 @@ void BtlLocalRecoValidation::analyze(const edm::Event& iEvent, const edm::EventS
 
     meHitEnergy_->Fill(recHit.energy());
     meHitTime_->Fill(recHit.time());
+    meHitTimeError_->Fill(recHit.timeError());
     meHitLongPos_->Fill(recHit.position());
     meHitLongPosErr_->Fill(recHit.positionError());
 
     meOccupancy_->Fill(global_point.z(), global_point.phi());
 
+    if (LocalPosDebug_) {
+      meLocalOccupancy_->Fill(local_point.x() + recHit.position(), local_point.y());
```

Comment:
@gsorrentino18 is the addition of the reconstructed longitudinal position giving the expected behaviour?

Reply:
The following occupancy plot shows that the BTL crystals are correctly arranged in the module, but there's still something that needs to be understood in the reconstruction of the longitudinal position inside the crystal.

<img width="947" alt="Schermata 2021-03-24 alle 09 38 51" src="https://user-images.githubusercontent.com/39314729/112284618-b630ed00-8c89-11eb-8542-eeaa475b6d6e.png">

Reply:
what is observed likely accounts for uncertainty in the reconstructed position that is used on top of the nominal center of the crystal according to the geometry, this is seen also in the one-dimensional plot of the longitudinal position in local coordinates

Reply:
in any case the 16x3 layout is confirmed to be in the correct orientation, unlike the ETL pixels layout (@parbol FYI)